backend/src/zango/core/monitoring/utils.py

```diff
@@ -10,42 +10,62 @@ def otel_is_enabled():
     """
     Returns the value of the `OTEL_IS_ENABLED` setting from the Django settings.
     """
-    return settings.OTEL_IS_ENABLED
+    return settings.OTEL_IS_ENABLED if hasattr(settings, "OTEL_IS_ENABLED") else False
```

Comment:
@deepakdinesh1123 just one small syntax change, can we change it to `getattr` ?
`getattr(settings, "OTEL_IS_ENABLED", False)`

please change it in all the places